tests/Model/BasePHPElement.php

```diff
@@ -52,35 +58,34 @@ protected function getFQN(Node $node): string
         return rtrim($fqn, "\\");
     }
 
-    protected static function convertReflectionTypeToString(?ReflectionType $type): string
+    protected static function getReflectionTypeAsArray(?ReflectionType $type): array
     {
+        $reflectionTypes = [];
         if ($type instanceof ReflectionNamedType) {
-            return (string)$type;
+            $type->allowsNull() && $type->getName() !== 'mixed' ?
+                array_push($reflectionTypes, '?' . $type->getName()) : array_push($reflectionTypes, $type->getName());
         }
         if ($type instanceof ReflectionUnionType) {
-            $reflectionType = '';
             foreach ($type->getTypes() as $type) {
-                $reflectionType .= (string)$type . '|';
+                array_push($reflectionTypes, $type->getName());
```

Comment:
Why not `array_push($reflectionTypes, (string) $type)`?

Reply:
The same as above